lib/menu.rb

```diff
@@ -0,0 +1,17 @@
+class Menu
+
+  attr_reader:list
+
+  def initialize
+    @list = {'fish':5.0,
+             'cake':2.99,
+             'beef':7.0}
+  end
+
+  def select_food(item)
+    @list.select { |key, value| key == item.to_sym }
```

Comment:
Lint/UnusedBlockArgument: Unused block argument - `value`. If it's necessary, use `_` or `_value` as an argument name to indicate that it won't be used.